origin-dapp/src/components/listing-detail.js

```diff
@@ -793,6 +793,7 @@ class ListingsDetail extends Component {
                 />
               )}
             </div>
+            {/* to-do: condition viewType based on category within for-rent schemaType rather than schemaType itself */}
```

Comment:
Does this mean that some for-rent listings will be fractional and some will be unit?

Reply:
No @jordajm but I suspected that various categories within for-rent would call for an hourly calendar while housing and maybe a few others would call for daily. 📆

Reply:
@jordajm can either the seller or buyer choose hourly vs daily? Or do we need to set it per-category for now? Some of them could realistically go either way.

Reply:
Good question. I think we could allow the seller to set hourly vs daily, but the buyer will have to book slots based on however the seller sets it up. I'll have to think through the technical side of letting the seller select what kind of calendar they want (and by extension what length of time constitutes a slot for their listing), but I think it makes a lot of sense. CC @wanderingstan

Reply:
Related: what's the max possible duration for each? Can an hourly selection be 25 hours or more? Can a daily selection be 365 days or more? Many of these categories might involve a couple of days but can otherwise be considered long, hourly rentals.

Reply:
I think it's really about the UX... 

If an hourly listing is longer than 24 hrs, the UX is going to get tricky just trying to let the user select all those hours. Same with a daily rental that goes longer than a month. 

I think we should probably lean towards hourly for `<24hrs`, daily for `>24hrs && <1month`, and something like [Luca's monthly booking UX](https://github.com/OriginProtocol/origin/issues/1006) for `>1month`

Reply:
Works for me 👍